app/javascript/shared/helpers/specs/MessageFormatter.spec.js

```diff
@@ -36,14 +36,18 @@ describe('#MessageFormatter', () => {
     it('should add links to @mentions', () => {
       const message =
         '@chatwootapp is an opensource tool thanks @longnonexistenttwitterusername';
-      expect(new MessageFormatter(message, true).formattedMessage).toMatch(
+      expect(
+        new MessageFormatter(message, false, true).formattedMessage
+      ).toMatch(
         '<p><a href="http://twitter.com/chatwootapp" target="_blank" rel="noreferrer nofollow noopener">@chatwootapp</a> is an opensource tool thanks @longnonexistenttwitterusername</p>'
       );
     });
 
     it('should add links to #tags', () => {
       const message = '#chatwootapp is an opensource tool';
-      expect(new MessageFormatter(message, true).formattedMessage).toMatch(
+      expect(
```

Comment:
Please add a test case for the new condition.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/4326/commits/3d8598b87489798ea4041ecb7a881099d3461e53